agent/lm_agent/tokenstat.py

```diff
@@ -209,6 +211,40 @@ async def attempt_tool_checks(
         return lri
 
 
+def get_all_product_features_from_cluster(show_lic_output) -> typing.List:
+    """
+    Returns a list of all product.feature in the cluster
+    """
+    PRODUCT_FEATURE = r"LicenseName=(?P<product>[a-zA-Z0-9]+)[_\-.](?P<feature>\w+)"
+    RX_PRODUCT_FEATURE = re.compile(PRODUCT_FEATURE)
+
+    parsed_features = []
+    output = show_lic_output.split("\n")
+    for line in output:
+        parsed_line = RX_PRODUCT_FEATURE.match(line)
+        if parsed_line:
+            parsed_data = parsed_line.groupdict()
+            product = parsed_data["product"]
+            feature = parsed_data["feature"]
+            parsed_features.append(f"{product}.{feature}")
+
+    return parsed_features
+
+
+def filter_entries_from_backend(entries, my_features):
```

Comment:
How do you feel about this function name instead?
```suggestion
def get_local_license_configurations(license_configurations, local_licenses):
```